index.html

```diff
@@ -6,6 +6,20 @@
     <meta name="viewport" content="width=device-width,initial-scale=1.0" />
     <link rel="icon" href="/favicon.ico" />
     <title>Cuttle</title>
+    <meta property="og:url" content="https://www.cuttle.cards/" />
+    <meta property="og:title" content="Cuttle" />
+    <meta property="og:type" content="website" />
+    <meta property="og:image" content="/public/img/cuttle-summary-banner.jpg/" />
```

Comment:
@Mal402 I gave you the wrong advice earlier. We should skip /public
```suggestion
    <meta property="og:image" content="/img/cuttle-summary-banner.jpg/" />
```